src/components/Image/ImageDropzone/MediaDropzone.tsx

```diff
@@ -55,6 +56,11 @@ export function MediaDropzone({
         constants.mediaUpload.maxVideoFileSize
       : settings.maxSize
     : constants.mediaUpload.maxVideoFileSize;
+
+  const seconds = settings?.maxVideoDuration ?? constants.mediaUpload.maxVideoDurationSeconds;
+  const durationLabel =seconds > 60 ? dayjs
+    .duration(seconds, 'seconds')
+    .format(`mm [minutes (${seconds} seconds)]`) : `${seconds} seconds`;
```

Comment:
I think we have a couple helper functions for this, try the `formatSeconds` from `number-helpers.ts`

Reply:
I know, but because I didn't want my task returned did it exactly as Matty wanted it 😬